param_templates/MOM_input.yaml

```diff
@@ -988,7 +990,9 @@ Global:
             If true, use a Laplacian horizontal viscosity."
         datatype: logical
         units: Boolean
-        value: True
+        value:
+            $OCN_GRID == "tx2_3v2": False
```

Comment:
Please revert LAPLACIAN to True. We will need it for the LEITHY backscatter, which will be introduced in an upcoming PR.

Reply:
done.